hardware-and-software-requirements.md

```diff
@@ -24,11 +24,22 @@ As an open source distributed NewSQL database with high performance, TiDB can be
 > - A large number of TiDB tests have been run on the CentOS 7.3 system, and in our community there are a lot of best practices in which TiDB is deployed on the Linux operating system. Therefore, it is recommended to deploy TiDB on CentOS 7.3 or later.
 > - The support for the Linux operating systems above includes the deployment and operation in physical servers as well as in major virtualized environments like VMware, KVM and XEN.
 > - Red Hat Enterprise Linux 8.0, CentOS 8 Stream, and Oracle Enterprise Linux 8.0 are not supported yet as the testing of these platforms is in progress.
-> - Support for CentOS 8 Linux is not planned because its upstream support ends on December 31, 2021.
+> - According to [CentOS Linux EOL](https://www.centos.org/centos-linux-eol/), the upstream support for CentOS ended on December 31, 2021.
```

Comment:
```suggestion
> - According to [CentOS Linux EOL](https://www.centos.org/centos-linux-eol/), the upstream support for CentOS Linux 8 ended on December 31, 2021. CentOS Stream 8 and CentOS Linux 7 continues to be supported.
```

See also https://github.com/pingcap/docs/pull/10310

Reply:
CentOS 8 and CentOS 7 are not verified by the QA team. We will add them here after the QA team has done the test.

Reply:
```suggestion
> - According to [CentOS Linux EOL](https://www.centos.org/centos-linux-eol/), the upstream support for CentOS Linux 8 ended on December 31, 2021. CentOS Linux 7 continues to be supported. We plan to support CentOS Stream 8 once testing has been completed.
```

- The text was unclear about the fact that the EOL is only for CentOS Linux 8. CentOS 7 continues to be supported upstream ( see https://wiki.centos.org/About/Product ). 
- This text is talking about upstream support for the OS, which directly influences support for TiDB as supporting TiDB on a OS that no longer has upstream support is a bad idea.
- We do support CentOS 7
- Support for TiDB on CentOS Stream 8 depends on testing by the QA team.